vision/src/autogluon/vision/predictor/predictor.py

```diff
@@ -152,7 +156,7 @@ def fit(self,
                 # This is the default preset in AutoGluon, but should generally only be used for quick prototyping.
```

Comment:
This is not the setting used by default. You should align the actual defaults with this setting.